README.md

```diff
@@ -87,7 +87,14 @@ script:
   - npm run test
 ```
 
+ - **docker**
```

Comment:
I think there is no need to add this item?

Reply:
I found the additional documentation useful for anyone encountering issues when running in docker, such as I did. Do you want me to delete this section in the Readme?